examples/text-generation/run_lm_eval.py

```diff
@@ -214,6 +214,32 @@ def _model_call(self, inps: torch.Tensor) -> torch.Tensor:
         logits = logits.to(torch.float32)
         return logits
 
+    def get_model_info(self) -> dict:
+        """
+        Mod from method to get Hugging Face model information for experiment reproducibility.
+        """
+
+        def get_model_num_params(model) -> int:
+            if hasattr(model, "num_parameters"):
+                return model.num_parameters()
+            if hasattr(model, "parameters"):
```

Comment:
```suggestion
            elif hasattr(model, "parameters"):
```

Used elif instead of a second if for better logical flow